### PR TITLE
Changes to hello_world.py

### PR DESCRIPTION
**Pull request by jovenchew**
For fun.

### CODE REVIEW

File: hello_world.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
 print("Hello World!")
 print("This is another change.")
+print("Even more changes")
```
Comment (cczhu) [inline hello_world.py]: Please add another line under this that says `print("Your text here")`.

File: .gitignore [hunk 2]
```diff
@@ -6,6 +6,9 @@ __pycache__/
 # C extensions
 *.so
 
+# vscode files
+.vscode/
+
```
Comment (cczhu) [inline .gitignore]: Good!

**Comment by jovenchew** [comment]
I have made the changes you requested.

**Comment by jovenchew** [comment]
I'm satisfied with the changes. Let's finish it.

**Pull request merged**